includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -837,13 +837,14 @@ public function get_payment_method_ids_enabled_at_checkout( $order_id = null ) {
 		$active_payment_methods     = $this->get_upe_enabled_payment_method_statuses();
 		foreach ( $capturable_payment_methods as $payment_method_id ) {
 			$payment_method_capability_key = $this->payment_method_capability_key_map[ $payment_method_id ] ?? 'undefined_capability_key';
-			if ( isset( $this->payment_methods[ $payment_method_id ] )
-				&& $this->payment_methods[ $payment_method_id ]->is_enabled_at_checkout( $order_id )
-				&& ( is_admin() || $this->payment_methods[ $payment_method_id ]->is_currency_valid() )
-				&& isset( $active_payment_methods[ $payment_method_capability_key ] )
-				&& 'active' === $active_payment_methods[ $payment_method_capability_key ]['status']
-			) {
-				$enabled_payment_methods[] = $payment_method_id;
+			if ( isset( $this->payment_methods[ $payment_method_id ] ) ) {
+				$processing_payment_method = $this->payment_methods[ $payment_method_id ];
+				if ( $processing_payment_method->is_enabled_at_checkout() && ( is_admin() || $processing_payment_method->is_currency_valid() ) ) {
```

Comment:
`$order_id` is dropped from the arguments of the `is_enabled_at_checkout`, is this intentional?

Reply:
Yep, the method [accepts no arguments](https://github.com/Automattic/woocommerce-payments/blob/develop/includes/payment-methods/class-upe-payment-method.php#L105).